src/plugins/intel_gpu/tests/test_cases/experimental_detectron_generate_proposals_single_image_gpu_test.cpp

```diff
@@ -177,76 +280,30 @@ struct experimental_detectron_generate_proposals_single_image_test
-
-
 TEST_P(experimental_detectron_generate_proposals_single_image_test_f32, basic) {
     ASSERT_NO_FATAL_FAILURE(test());
 }
-
-TEST_P(experimental_detectron_generate_proposals_single_image_test_f16, basic) {
-    ASSERT_NO_FATAL_FAILURE(test());
-}
-
-template <typename T>
-std::vector<ExperimentalDetectronGenerateProposalsSingleImageParams<T>> getExperimentalDetectronGenerateProposalsSingleImageParams() {
-    std::vector<ExperimentalDetectronGenerateProposalsSingleImageParams<T>> params = {
-            {
-                    0.0f, 0.7f, 1000, 6,
-                    getValues<T>({149.0, 149.0, 149.0, 149.0,
-                     149.0, 0.0, 149.0, 149.0,
-                     149.0, 60.8744, 149.0, 149.0,
-                     149.0, 61.8950, 149.0, 149.0,
-                     149.0, 149.0, 149.0, 149.0,
-                     149.0, 149.0, 149.0, 149.0}),
-                    getValues<T>({0.989487, 0.942969, 0.940246, 0.923749, 0.905361, 0.884219})
-            },
-            {
-                    1.5f, 0.4f, 1000, 10,
-                    getValues<T>({43.171, 0.31823, 53.5592, 149,
-                     0, 75.2272, 149, 87.2278,
-                     141.058, 114.876, 149, 149,
-                     0, 146.297, 149, 149,
-                     0.0, 0.0, 0.0, 0.0,
-                     0.0, 0.0, 0.0, 0.0,
-                     0.0, 0.0, 0.0, 0.0,
-                     0.0, 0.0, 0.0, 0.0,
-                     0.0, 0.0, 0.0, 0.0,
-                     0.0, 0.0, 0.0, 0.0,
-                     0.0, 0.0, 0.0, 0.0}),
-                    getValues<T>({0.695249, 0.411288, 0.0941284, 0.0794298, 0.0,
-                     0.0, 0.0, 0.0, 0.0, 0.0})
-            },
-            {
-                    5.0f, 0.71f, 10, 15,
-                    getValues<T>({43.171, 0.31823, 53.5592, 149,
-                     0, 75.2272, 149, 87.2278,
-                     141.058, 114.876, 149, 149,
-                     149, 149, 149, 149,
-                     30.2866, 149, 149, 149,
-                     149, 149, 149, 149,
-                     149, 126.679, 149, 149,
-                     149, 6.53844, 149, 149,
-                     149, 0, 149, 149,
-                     149, 149, 149, 149,
-                     0, 0, 0, 0,
-                     0, 0, 0, 0,
-                     0, 0, 0, 0,
-                     0, 0, 0, 0,
-                     0, 0, 0, 0}),
-                    getValues<T>({0.695249, 0.411288, 0.0941284, 0.0, 0.0, 0.0,
-                     0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0})
-            }
-    };
-    return params;
-}
-
 INSTANTIATE_TEST_SUITE_P(
         experimental_detectron_generate_proposals_single_image_gpu_test,
         experimental_detectron_generate_proposals_single_image_test_f32,
-        ::testing::ValuesIn(getExperimentalDetectronGenerateProposalsSingleImageParams<float>()));
+        ::testing::Combine(
+                ::testing::ValuesIn(getExperimentalDetectronGenerateProposalsSingleImageParams<float>()),
```

Comment:
Please not to use Combine for creating test as much as possible. Instead, please try to pick meaningful test instances, i.e., F > 16 is preferable to test fsv16 and B>16 is preferable to test bsv32.

Reply:
This operation has nothing to do with batches and features - it processes single-image inputs (see https://docs.openvino.ai/latest/openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_6.html ).

Regarding number of tests - agree, reduced it in last commit. 
Also added the operation to program.cpp.